src/model.rs

```diff
@@ -1172,6 +1212,7 @@ impl Model {
     /// ```
     pub fn new(mut c: Collections) -> Result<Self> {
         c.comment_deduplication();
+        c.clean_comments();
```

Comment:
Current "comment_deduplication" would handle multiple things, but for performances it's probably way better to do it inside `comment_deduplication` (and rename?).
If so, to b future-proof, please keep in mind that the key for comment comparison might change (and contain more than just `comment.name` in the future) :wink: